src/gmail.js

```diff
@@ -17,14 +17,10 @@ var Gmail = function(localJQuery) {
         $ = localJQuery;
     } else if (typeof jQuery !== "undefined") {
         $ = jQuery;
+    } else if (localJQuery === false) {
+        // leave $ undefined, which may be fine for some purposes.
```

Comment:
This is my fault really, but this code needs to be tweaked slightly.

We need this check before `if (typeof localJQuery !== "undefined") {` otherwise this code will never trigged.